storage-bigtable/src/access_token.rs

```diff
@@ -57,6 +59,8 @@ impl AccessToken {
                 credentials,
                 scope,
                 token,
+                token_refresh_start_time: Arc::new(AtomicU64::new(0)),
+                get_token_timeout_seconds: 5,
```

Comment:
I think this is a very optimistic timeout and personally have increase this a bit.